src/lib/sql/sql_translator.cpp

```diff
@@ -1363,9 +1364,61 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_create_table(const hs
 
       column_definition.name = parser_column_definition->name;
       column_definition.nullable = parser_column_definition->nullable;
+
+      // Translate column constraints
+      for (const auto& column_constraint : *parser_column_definition->column_constraints) {
+        if (column_constraint != hsql::ConstraintType::Unique &&
+            column_constraint != hsql::ConstraintType::PrimaryKey) {
+          continue;
+        }
+        const auto constraint_type = column_constraint == hsql::ConstraintType::PrimaryKey
+                                         ? KeyConstraintType::PRIMARY_KEY
+                                         : KeyConstraintType::UNIQUE;
+        table_key_constraints.emplace_back(std::unordered_set<ColumnID>{column_id}, constraint_type);
```

Comment:
As I saw the line `const auto unique_constraint = TableKeyConstraint{{ColumnID{0}}, KeyConstraintType::UNIQUE};` down below, maybe we can use `emplace_back({ColumnID{0}}, constraint_type)`?

Reply:
Unfortunately, this does not work (`column_id` is of type `ColumnID`)
```
/Users/deyass/Documents/phd/hyrise/src/lib/sql/sql_translator.cpp:1377:31: error: no matching member function for call to 'emplace_back'
        table_key_constraints.emplace_back({column_id}, constraint_type);
        ~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
/opt/homebrew/bin/../include/c++/v1/vector:721:19: note: candidate template ignored: substitution failure: deduced incomplete pack <(no value), const opossum::KeyConstraintType &> for template parameter '_Args'
        reference emplace_back(_Args&&... __args);
```